packages/app-desktop/gui/MainScreen/MainScreen.tsx

```diff
@@ -639,6 +650,12 @@ class MainScreenComponent extends React.Component<Props, State> {
 				_('Set the password'),
 				onViewEncryptionConfigScreen
 			);
+		} else if (this.props.showInstallTemplatesPlugin) {
+			msg = this.renderNotificationMessage(
+				_('Templates feature is now moved to a plugin called "Templates".'),
```

Comment:
> The template feature has been moved to a plugin called "Templates"

Please don't translate as it's temporary (remove the `_()`)

Reply:
done